packages/amplify-appsync-simulator/package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "rimraf ./lib tsconfig.tsbuildinfo tsconfig.tests.tsbuildinfo node_modules",
     "watch": "tsc -w",
     "start": "node ./lib/index.js",
-    "test": "jest"
+    "test": "jest --forceExit"
```

Comment:
why is this needed, seems to be more of a [escape hatch](https://jestjs.io/docs/cli#--forceexit)

Reply:
Actually the unit tests terminate graciously on my machine but in CI they pass but for some reason fail to exit. I have tried to use `jest --detectOpenHandles` to see what's going on but even that doesn't give me any info. So, had to add this flag like in the mock plugin tests.